index.html

```diff
@@ -18,19 +18,24 @@
 
   <h1>Learn to code by watching others</h1>
 
-  See how experienced developers solve problems in real-time. Watching scripted tutorials is great, 
-  but understanding how developers think is invaluable. 
+  <p>See how experienced developers solve problems in real-time. Watching scripted tutorials is great, 
+  but understanding how developers think is invaluable. </p>
 
-  Try it free 7 days then $20/mo. thereafter
 
-  First Name 
-  Last Name 
-  Email Address
-  Password
+  <div>
+    <span>Try it free 7 days then $20/mo. thereafter</span>
+  </div>
```

Comment:
Maybe encapsulate the h1 to this div in a section / header as well? Because we will need to change the direction of this content and the form between the mobile and desktop layouts.

Reply:
I didn't understand which div you're referring to